nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp delete comment

Description:
Backend code for creating comments is complete.

Testing Instructions:

Postman URL:
```
http://localhost:8000/comments/1
```

Make a DELETE request in Postman and check the database to see if the comment with id = 1 is deleted.
